api/src/main/resources/beans_2_0.xsd

```diff
@@ -353,5 +354,14 @@
       </xs:choice>
     </xs:complexType>
   </xs:element>
+  <xs:element name="trim" type="xs:string" fixed="">
```

Comment:
Yes I was thinking about this as well. So then it makes sense to change it as above. I don't think it makes sense to allow multiple occurences of e.g `<scan>`.  

I don't know xsd much, but now each of these elements could be specified only once or do somebody have better idea how to define this???